src/test/java/org/broadinstitute/hellbender/utils/pileup/PileupElementUnitTest.java

```diff
@@ -147,6 +147,8 @@ public void testPrevAndNextTest(final GATKRead read, final CigarOperator firstOp
         Assert.assertEquals(pe.getBetweenPrevPosition().size(), 0);
         assertEqualsOperators(pe.getBetweenNextPosition(), ops);
         Assert.assertEquals(pe.getPreviousOnGenomeCigarElement(), null);
+        Assert.assertEquals(pe.getNeighborNotOnGenomeOperator(PileupElement.Direction.PREV), null);
+        Assert.assertEquals(pe.getNeighborNotOnGenomeOperator(PileupElement.Direction.NEXT), ops.get(0));
```

Comment:
The neighboring "not on genome" operator is always the first element in ops? What if `ops.get(0)` is an "on genome" operator?